packages/react-reconciler/src/ReactFiberWorkLoop.new.js

```diff
@@ -426,6 +431,13 @@ export function requestUpdateLane(fiber: Fiber): Lane {
   // TODO: Move this type conversion to the event priority module.
   const updateLane: Lane = (getCurrentUpdatePriority(): any);
   if (updateLane !== NoLane) {
+    if (
+      enableSyncDefaultUpdates &&
+      (updateLane === InputContinuousLane ||
+        updateLane === InputContinuousHydrationLane)
+    ) {
+      return DefaultLane;
```

Comment:
Need to do the same check for `eventLane` below. Probably should have a test for this, too.